core/src/main/scala/stryker4s/run/report/Reporter.scala

```diff
@@ -1,13 +1,27 @@
 package stryker4s.run.report
+import grizzled.slf4j.Logging
 import stryker4s.config.Config
 import stryker4s.model.MutantRunResults
 
-class Reporter {
+class Reporter extends Logging{
 
   /**
     * Generate a report for each reporter that is available.
     */
   def report(runResult: MutantRunResults)(implicit config: Config): Unit = {
     config.reporters.foreach { _.report(runResult) }
   }
+
+  def determineExitCode(runResults: MutantRunResults)(implicit config: Config): Int = {
```

Comment:
Not sure if determining an exit code really is the responsibility of a reporter. I think I'd prefer to have a separate class for it